README.md

```diff
@@ -25,6 +53,27 @@ minikube start -p mk-tekton \
 eval $(minikube docker-env -p mk-tekton)
 ```
 
+### Development build
+
+1. change directory to '${GOPATH}/src/github.com/openshift/tektoncd-pipeline-operator'
+```
+cd ${GOPATH}/src/github.com/openshift/tektoncd-pipeline-operator
+```
+2. go build
+``` 
+CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o build/_output/bin/tekton-pipeline-operator ./cmd/manager/
```

Comment:
it is better to use the operator-sdk itself to do the build e.g. `operator-sdk build <image-name>`

Reply:
Thanks for review.  I updated to use the operator-sdk